FlowCal/excel_ui.py

```diff
@@ -1489,9 +1490,32 @@ def run(input_path=None,
     if verbose:
         print("\nDone.")
 
-if __name__ == '__main__':
-    # Read command line arguments
+
+def run_command_line(argv=None):
+    """
+    Entry point for the FlowCal and flowcal console scripts
+
+    Parameters
+    ----------
+    argv: list of strings
+        argument values to parse and pass to run()
+
+    Notes
+    ----------
+    Use main(argv = ['arguments', 'you', 'want', 'to', 'test'])
+    to test entry point and argument parsing in unit tests
+
+    See Also
+    ----------
+    FlowCal.excel_ui.run()
+
+    http://amir.rachum.com/blog/2017/07/28/python-entry-points/
+
+    """
+    if argv == None:
```

Comment:
Should be `is None`. Check https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none